dev-build/conf/routes

```diff
@@ -120,6 +120,9 @@ GET            /video/end-slate/series/*seriesId.json
 GET            /video/end-slate/section/*sectionId.json                                                                          controllers.VideoEndSlateController.renderSection(sectionId)
 GET            /embed/card/*path.json                                                                                            controllers.RichLinkController.render(path)
 GET            /embed/card/*path                                                                                                 controllers.RichLinkController.renderHtml(path)
+GET            /embed/contentcard/*path.json                                                                                     controllers.ContentCardController.render(path, group: Option[Int], card: Option[Int])
```

Comment:
Could we rename the controller `ContentCardController` to something a bit more descriptive?

The above two routes aren't descriptive either, but the controller is `RichLinkController` describes the feature which helps.